server/migrations/20210225221835-add-clover-finance.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkInsert('Chains', [{
+        base: 'substrate',
+        id: 'clover',
+        symbol: 'CLV',
+        name: 'Clover Finance',
+        icon_url: '/static/img/protocols/clover.png',
+        type: 'chain',
+        network: 'clover',
+        active: true,
+        description: 'An open cross-chain bridge protocol based on Substrate.',
+        telegram: 'https://t.me/clover_en',
+        website: 'https://clover.finance/',
+        github: 'https://github.com/clover-network',
+        collapsed_on_homepage: false,
+      }], { transaction: t });
+
+      await queryInterface.bulkInsert('ChainNodes', [{
+        chain: 'clover',
+        url: 'wss://api.clover.finance/',
```

Comment:
It does seem like it's currently not working because of `wss` being down on Clover's end, but I also recall that `ws` gives an error due to being insecure. Not sure what to do about this.

Reply:
Never mind, it seems to be working with `ws`